### DISCUSSION TITLE
How can i insert in an inline link(link address is www.google.com, for example) to a property content(which is rich text datatype, not url) of a page inside a database?

### DISCUSSION THREAD

**Discussion by tyehuoPHD** [jheddings/notional#133] [General]
# How can i insert in an inline link(link address is www.google.com, for example) to a property content(which is rich text datatype, not url) of a page inside a database?

i fully understand how to create page, update page property content of pages inside databases.
But, i dont know how to insert inline url link to certain text inside page property content.

**Comment by jheddings** [comment]
First, you will need to build the rich text property value from an array of rich text objects:

```python
awesome = types.RichText[
    text.TextObject["visit my "],
    text.TextObject["awesome", "https://youtu.be/a0VGJYAMMGg"],
    text.TextObject[" website!"],
]
```

Then, you can update the page property using either the API:

```python
page = notion.pages.retrieve(page_id)
notion.pages.update(page, Summary=awesome)
```

Or using an ORM class:

```python
class AwesomeData(CustomPage):
    __database__ = MY_AWESOME_DATABASE_ID

    Title = Property("Title", schema.Title())
    Summary = Property("Summary")

page = notion.pages.retrieve(page_id)
record = AwesomeData(page)
record.Summary = awesome
```

**Comment by tyehuoPHD** [comment]
thanks for the help
